examples/kubeval/README.md

```diff
@@ -0,0 +1,26 @@
+# Kubeval
+
+The `kubeval` KRM config function validates Kubernetes resources against their
```

Comment:
nit: Remove unnecessary filler:

`Validates Kubernetes resources against their Kubernetes OpenAPI definition using [kubeval]`

Link to kubeval docs.

Reply:
Done